quil-rs/src/program/scheduling/graph.rs

```diff
@@ -869,6 +875,20 @@ NONBLOCKING CAPTURE 0 "ro_rx" flat(duration: 2.0000000000000003e-06, iq: 1.0, sc
 JUMP-WHEN @eq ro
 LABEL @eq
 PULSE 0 "ro_tx" gaussian(duration: 1, fwhm: 2, t0: 3)
+"#
+    }
+
+    build_dot_format_snapshot_test_case! {
```

Comment:
I'd just leave a comment to clarify intent. Something like:

> Note, as of this commit there is no support for memory dependencies across blocks; i.e. in this example there is no await capture edge from the `CAPTURE` to the `MOVE` instruction. Upon support for this feature, we can simply update the snapshot.

Reply:
This is documented for `ScheduledProgram` in general now, and I don't think we'll ever support that feature